common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -99,3 +99,18 @@
 else:
     tmpl.render_context(context)
 %></%def>
+
+<%def name="optional_include_mako_with_fallback(file, fallback, with_microsite=False, **kwargs)"><%
```

Comment:
@singingwolfboy do you suggest to extend optional_include_mako function with fallback and kwargs params? It seemed that in slack discussion you first offered to create separated function like optional_include_mako_with_fallback with additional arguments.
Ok I will extend existed function with new arguments.

> Also, what happens if the template author wants to define a list of fallbacks?
Do you suggest to provide fallback param as a list of pathes to templates? I can do it but I think that one fallback template will be sufficient for the 99% cases.